README.md

```diff
@@ -46,7 +46,7 @@ Ozone FOSS requires you to log into each component separately:
 ## 3. Analytics in Ozone FOSS
 There is a number of manual steps required to refresh analytics data in Ozone FOSS, please follow [this](readme/analytics.md) guide to know more.
 
-:bulb: **Did you know?** Ozone Pro embeds a service called *Ozone Analytics* that streams data from all its components automatically and in real-time data to any reporting and BI platform.
+:bulb: **Did you know?** Ozone Pro embeds a service called *Ozone Analytics* that streams data from all its components automatically and in real-time transfers data to any reporting and BI platform.
```

Comment:
`Self review:` I suppose the Ozone Analytics after streaming data from all it's component, it then **transfers** the data to any reporting and BI platform